### PR TITLE
[EPMGCIPART-403] Update User Profile Data for ArtObject, Proposal, and Tender

### PR DESCRIPTION
**Pull request by aborozdykh**

### CODE REVIEW

File: src/main/java/com/artograd/api/helpers/UserServiceHelper.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  public void updateUserProfileDataInArtObjects(String userName, List<UserAttribute> attributes) {
+
+    artObjectRepository
+        .findByOwnerIdOrSupplierId(userName)
+        .forEach(
+            ao -> {
+              if (ao.getOwner() != null && ao.getOwner().getId().equals(userName)) {
+                ao.getOwner().setName(userAttributeHelper.formatUserName(attributes));
+                ao.getOwner()
+                    .setPicture(userAttributeHelper.getUserAttributeValue(attributes, "picture"));
```
Comment (Maksim-Totski) [inline src/main/java/com/artograd/api/helpers/UserServiceHelper.java:31]: Please refer to ENUM value from com.artograd.api.model.enums.UserAttributeKey

File: src/main/java/com/artograd/api/helpers/UserServiceHelper.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                ao.getOwner()
+                    .setOrganization(
+                        userAttributeHelper.getUserAttributeValue(
+                            attributes, "custom:organization"));
```
Comment (Maksim-Totski) [inline src/main/java/com/artograd/api/helpers/UserServiceHelper.java:35]: same

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -23,7 +23,7 @@ aws:
   cognito:
     userPoolId: ${AWS_COGNITO_USER_POOL_ID}
   sqs:
-    mails: ${SQS_EMAILS}
+    mails: ${SQS_EMAILS:https://sqs.eu-central-1.amazonaws.com/124092974016/dev_emails.fifo}
```
Comment (Maksim-Totski) [inline src/main/resources/application.yaml:26]: let it stay in env vars

**Review by Maksim-Totski — CHANGES_REQUESTED** [review]

File: src/main/java/com/artograd/api/helpers/UserServiceHelper.java (hunk 1 above)
Comment (aborozdykh) [reply src/main/java/com/artograd/api/helpers/UserServiceHelper.java]: Done

File: src/main/java/com/artograd/api/helpers/UserServiceHelper.java (hunk 2 above)
Comment (aborozdykh) [reply src/main/java/com/artograd/api/helpers/UserServiceHelper.java]: Done.

**Review by Maksim-Totski — APPROVED** [review]

**Pull request merged**